src/release/breaking-changes/win_lifecycle_process_function.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Introduction of FlutterEngine::ProcessExternalWindowMessage
+description: External windows should call ProcessExternalWindowMessage to be considered for application lifecycle events
+---
+
+## Summary
+
+In order to be included in the logic for app lifecycle on Windows, any
+external windows added to a Flutter app should invoke
+`FlutterEngine::ProcessExternalWindowMessage` from its `WndProc` function.
+

```

Comment:
I would be tempted to add a "Who is affected" section here.

Reply:
Please add this section